draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1272,6 +1272,18 @@ time period considered appropriate (e.g., randomize `iat` within the last 24
 hours and calculate `exp` accordingly) or rounded (e.g., rounded down to the
 beginning of the day).
 
+SD-JWT only conceals the value of claims that are not revealed.
+It does not meet standard security notations for anonymous credentials. In
```

Comment:
Can we have a reference to what "anonymous credentials"? do we really expect all of the readers to know what it is? this is the first time that they are mentioned in the text I believe?

Reply:
I think the idea was to add a citation to CL01, but that might not actually have the phrase in it easily findable...

Reply:
I added the citation below.